src/libpmemobj/memblock.c

```diff
@@ -471,10 +472,15 @@ run_get_bitmap(const struct memory_block *m, struct run_bitmap *b)
 	struct chunk_header *hdr = heap_get_chunk_hdr(m->heap, m);
```

Comment:
if you have cached bitmap, you don't need header - you can move this line to "else branch"